sites/footers/includes/footer.html

```diff
@@ -1,65 +1,130 @@
+{%- capture contextualFooterTitle -%}
+	{%- if page.contextualFooter.title -%}
+		{{ page.contextualFooter.title }}
+	{%- elsif site.global.contextualFooter.title[i18nText-lang] -%}
+		{{ site.global.contextualFooter.title[i18nText-lang] }}
+	{%- endif -%}
+{%- endcapture -%}
+
 <footer id="wb-info">
+	<h2 class="wb-inv">{{ i18nText-footerSite }}</h2>
+{% unless page.noFooterContextual %}
+{%- if contextualFooterTitle != "" and site.global.contextualFooter.links[i18nText-lang] or page.contextualFooter.links -%}
+	<div class="gc-contextual">
+		<nav class="container">
```

Comment:
Should we de-associate the layout with the semantic or can we tolerate here? I am questioning because the footer brand do that de-association. We should go for consistency. @GormFrank what do you think?

For example, it will look like:
```suggestion
		<div class="container"><nav>
```

---

Or we can look to combine the `div.container` with the `div.brand-row` div elements. so we will have something like: `div.container.d-flex.align-items-center`

Reply:
I should point out that the two other sections of the footer do not have elements not associated with the nav. In the brand section, the Canada.ca logo is outside the scope of the nav. Not sure if it's big deal or not.

However, I agree that we should de-associate the layout from the semantic. I am god with your suggested change.